packages/bot-web-ui/src/utils/validator/index.js

```diff
@@ -1 +1 @@
-export default from './validator';
+export default from './validator.js';
```

Comment:
Is this change expected as part of this PR?

Reply:
it is required cause if i didn't include .js the test won't pass